Production/interface/Selectors.h

```diff
@@ -15,8 +15,17 @@ struct TagObject {
     int charge;
     unsigned id;
     float isolation;
+    bool extramuon;
+    bool extraelectron;
+    bool dimuon;
 };
 
+bool muonveto (const std::vector<pat::Muon>& muons, const pat::Muon *ref_muon, const reco::Vertex& primaryVertex);
+
+bool electronveto (const std::vector<pat::Electron>& electrons, const reco::Vertex& primaryVertex, const float rho);
+
+bool dimuonveto (const std::vector<pat::Muon>& muons, const pat::Muon *ref_muon, const reco::Vertex& primaryVertex);
```

Comment:
- `ref_muon` is not needed for dimionveto
- better to rename functions to `hasExtraMuon` etc.
- define functions directly in an anonymous namespace in Selectors.cc